### PR TITLE
turn on prefer-const

### PR DESCRIPTION
**Pull request by chrisjshull**
per https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -338,7 +338,7 @@ module.exports = {
     'no-var': 2,
     // 'object-shorthand': 0,
     // 'prefer-arrow-callback': 0,
-    // 'prefer-const': 0,
+    'prefer-const': ["error", {"destructuring": "all"}],
```
Comment (philipwalton) [inline index.js]: Switch this to single quotes so the tests pass.

File: index.js [hunk 2]
```diff
@@ -338,7 +338,7 @@ module.exports = {
     'no-var': 2,
     // 'object-shorthand': 0,
     // 'prefer-arrow-callback': 0,
-    // 'prefer-const': 0,
+    'prefer-const': ['error', {'destructuring': 'all'}],
```
Comment (philipwalton) [inline index.js]: Sorry, should have also mentioned, can you remove the quotes from the object key `destructuring`? It should be:

```js
'prefer-const': ['error', {destructuring: 'all'}],
```

Comment (chrisjshull) [reply index.js]: You know, I have "allow edits from maintainers" turned on ;)

Comment (philipwalton) [reply index.js]: I did not know that! But it all looks good now anyway, thanks!

**Pull request merged**